AbstractAlgebraInLean/Exercise/Ex13.lean

```diff
@@ -0,0 +1,27 @@
+import Mathlib.Tactic
+import Mathlib.Algebra.Group.Basic
+import Mathlib.Init.Function
+-- Suppose G is a group, a is an element in G.
+variable  [Group G] (a : G)
+-- Define f x that equals to a*x.
+def f (x : G) : G := a * x
+-- To prove f is bijective.
+example : (f a).Bijective := by
+  constructor
+  -- To prove f is injective.
+  ·-- f x = f y ,that is a*x = a*y , implies x = y by mul_left_cancel.
```

Comment:
-- Suppose we have x y ∈ G, f x = f y,
intro x y h
-- which is a * x = a * y.
rw [f] at h
rw [f] at h
-- Thus we have x = y since we can cancel a in the left.
apply mul_left_cancel h